domino_data/configuration_gen.py

```diff
@@ -90,6 +92,7 @@ class CredElem(Enum):
     USERNAME = "username"
     CLIENTID = "clientId"
     CLIENTSECRET = "clientSecret"
+    PERSONALACCESSTOKEN = "personalAccessToken"
```

Comment:
why is it `personalAccessToken` here and `personalToken` in the openapi file?

Reply:
It's because the field is called `personalAccessToken` but the name is called `personalToken` in the config for data-services. Would this cause any problems?

Reply:
<img width="284" alt="Screenshot 2023-09-11 at 3 52 03 PM" src="https://github.com/dominodatalab/domino-data/assets/140661003/0ab0b592-5a0d-4224-ba94-cca329a643c4">

Reply:
not that's good, thanks